### ISSUE TITLE
NTR: Abdominal imaging: liver rupture (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8991]
New term request Abdominal imaging: liver rupture (elthat_110514024625): MRI; CT; hepatic  UMLS: 0341444

**Comment by pnrobinson** [comment]
@MickeySegal
even though this sounds straightforward, it seems to be difficult to find a specific definition.
The DD includeds HELLP and trauma, and what seems to be observed is 
"Subcapsular hepatic rupture is characterized by the development of a hematoma, often quite large, beneath the liver capsule. An interval of days or hours may elapse between the time of injury and the first appearance of symptoms. A history of injury may be entirely lacking, or may be so unremarkable that it is elicited only by diligent interrogation. The hematoma tends to increase in size, often progressing to the point of rupture." (note the definition is circular and does not really make sense -- from 1962 (https://jamanetwork.com/journals/jama/article-abstract/332572).
However, I am wondering if this is supposed to refer to HELLP rather than say knife injury, whether we should call the feature
Subcapsular liver hematoma 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7516696/

**Comment by MickeySegal** [comment]
Other uses we cite:

https://www.ncbi.nlm.nih.gov/books/NBK1494/
"Other rare complications include rupture of the spleen or liver"

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4674738/
"Even the possibility of hepatic rupture varies in percentage considering the different sites of the disease: on the right liver in 75% of the cases, on the left liver in 11% of the cases, and on both in 14% of the cases"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000998
- label: Liver rupture
- parents: [Abnormal liver morphology (HP:0410042)](https://hpo.jax.org/app/browse/term/HP:0410042)
- pmid: PMID:13915569
- def: Subcapsular hepatic rupture is characterized by the development of a hematoma, often quite large, beneath the liver capsule. An interval of days or hours may elapse between the time of injury and the first appearance of symptoms. A history of injury may be entirely lacking, or may be so unremarkable that it is elicited only by diligent interrogation. The hematoma tends to increase in size.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8991
- synonyms:
